package.json

```diff
@@ -303,7 +303,7 @@
     "eslint-config-prettier": "6.11.0",
     "eslint-plugin-graphql": "3.1.1",
     "eslint-plugin-react": "7.19.0",
-    "eslint-plugin-react-hooks": "3.0.0",
+    "eslint-plugin-react-hooks": "^4.0.4",
```

Comment:
Hah. Not exactly sure what prompted that change. I just did regular `yarn install`...